cucumber/authenticators_jwt/features/authn_jwt_check_standard_claims.feature

```diff
@@ -0,0 +1,367 @@
+    """
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt "issuer" variable to value "incorrect-value"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with raw service ID
+    Then host "myapp" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
+    """
+
+  Scenario: JWT token with past exp claim value, 401 Error
+    Given I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: jwks-uri
+    """
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject",
+      "exp": 0
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with raw service ID
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00016E Token expired
+    """
+
+  Scenario: Valid JWT token with no exp claim, 401 Error
+    Given I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: jwks-uri
+    """
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I issue a JWT token without exp:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with raw service ID
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00091E Failed to validate token, mandatory claim 'exp' is missing.
+    """
+
+  Scenario: JWT token with future iat claim, 401 Error
+    Given I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: jwks-uri
+    """
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject",
+      "iat": 7624377528
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with raw service ID
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00035E Failed to decode token (3rdPartyError ='#<JWT::InvalidIatError: Invalid iat
```

Comment:
looks like the message cut in the middle..its o.k.?

Reply:
FIxed to 
CONJ00035E Failed to decode token (3rdPartyError ='#<JWT::InvalidIatError: Invalid iat>')>